### PR TITLE
Add Nationality Field

### PR DESCRIPTION
**Pull request by maj0-0**
Haven't worked on testcases, will work on it one time after finishing gender field

### CODE REVIEW

**Review by Bearypop — COMMENTED** [review]
LGTM

**Review by neyapraveen — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -23,29 +24,35 @@ public class SampleDataUtil {
...
+                getSocialMediaLinkSet("https://example.com/irfan"),
+                new Nationality("local")),
             new Person(new Name("Roy Balakrishnan"), new Major("Computer Science"), new Year("2"),
-                    new Email("royb@u.nus.edu"), new Description("Colleague from work"),
-                    getTutorialSet("11", "19", "22"),
-                    getSocialMediaLinkSet("https://example.com/roy"))
+                new Email("royb@u.nus.edu"), new Description("Colleague from work"),
+                getTutorialSet("11", "19", "22"),
+                getSocialMediaLinkSet("https://example.com/roy"),
+                new Nationality("foreigner"))
```
Comment (wnchan) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:55]: LGTM!

**Review by wnchan — COMMENTED** [review]
LGTM!

**Review by PearlynnT — COMMENTED** [review]
Might want to merge this first before the gender field branch so that there's no need to copy and paste the test case code

**Comment by maj0-0** [comment]
Nationality field implementation is in the Gender PR as well, so closing this one cos its a duplicate

**Pull request closed without merging**